torch/_inductor/pattern_matcher.py

```diff
@@ -598,6 +598,224 @@ def addmm(match, mat1, mat2, inp):
         return L[aten.add](inp, L[aten.mm](mat1, mat2))
 
 
+if torch._C.has_mkldnn:
+    mkldnn = torch.ops.mkldnn
+    # _conv_args = (Arg(), Arg(), Arg(), Arg(), Arg(), Arg(), Arg(), KeywordArg('attr'), Arg(),  Arg(),)
+    _conv_args = (Arg(), Arg(), Arg(), Arg(), Arg(), Arg(), Arg())
+    _user_1 = [CallFunction(aten.mkldnn_convolution, *_conv_args, _users=1)]
+    _user_2 = [CallFunction(aten.mkldnn_convolution, *_conv_args, _users=2)]
+    _user_3 = [CallFunction(aten.mkldnn_convolution, *_conv_args, _users=3)]
+    _user_4 = [CallFunction(aten.mkldnn_convolution, *_conv_args, _users=4)]
```

Comment:
I'm thinking of adding a utility function to deduce the value of `_users` directly, as Jiong suggested. The `_user_x` is confusing.

Reply:
changed.